File: python/remove_duplicates_from_sorted_array.py
class Solution:
    # @param a list of integers
    # @return an integer
    def removeDuplicates(self, A):
        if len(A) > 1:
            length = 0
            for index in range(1,len(A)):
                if A[length] != A[index]:
                    length += 1
                    A[length] = A[index]
            length += 1
        else:
            length = len(A)
        return length

File: cpp/remove_duplicates_from_sorted_array.cpp
class Solution {
public:
    int removeDuplicates(int A[], int n) {
        int length = 0;
        if ( n == 0 || n==1)
            return n;
        else
        {
            for(int i = 1; i<n; i++)
            {
                if (A[i] != A[length])
                {
                    A[++length] = A[i];
                }
            }
            return length + 1;
        }

    }
};